Wikipedia/Categories/NSAttributedString+WMFModifyParagraphs.m

```diff
@@ -0,0 +1,25 @@
+//  Created by Monte Hurd on 8/17/15.
+//  Copyright (c) 2015 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
+
+#import "NSAttributedString+WMFModifyParagraphs.h"
+
+@implementation NSAttributedString (WMFModifyParagraphs)
+
+- (NSAttributedString*)wmf_attributedStringWithParagraphStylesAdjustments:(void (^)(NSMutableParagraphStyle* existingParagraphStyle))block {
+    if (!block) {
+        return self;
+    } else {
+        NSMutableAttributedString* mutableCopy = self.mutableCopy;
+        [self enumerateAttribute:NSParagraphStyleAttributeName
```

Comment:
I thought this too, but it looks because we're changing the paragraph for the exact same ranges, it actually replaces the old paragraph objs for those ranges. I confirmed by setting a breakpoint and looking at the string before and after. Want to double check? Or do you think I should add a test to confirm?